public/503.html

```diff
@@ -92,11 +92,11 @@ <h3 class="landmark heading">Site Navigation</h3>
       <div class="clear"></div>
     </div>
     <!-- END header -->
-    <div id="main" class="system error 503 maintenance">
-      <h2>Error 503 - Service unavailable</h2>
-      <h3>Exceeded maximum allowed downloads per minute.</h3>
-      <p>To combat bots, we are currently banning IP addresses that download too many works in a short time period. If you see this page repeatedly please pause at least 10 seconds between downloads. If you are banned, you will be unable to access the Archive. Access will be restored 24 hours after the ban started.</p>
-      <p>Follow <a href="http://twitter.com/ao3_status/">@AO3_Status</a> on twitter for updates.</p>
+    <div id="main" class="system error 503">
```

Comment:
Yes I did it in each of the files.
            <li><a href="/archive_faqs">FAQ</a></li>
            <li><a href="/wrangling_guidelines">Wrangling Guidelines</a></li>
            <li><a href="/donate">Donate or Volunteer</a></li>